project/NativeImage.scala

```diff
@@ -86,7 +86,7 @@ object NativeImage {
     initializeAtRuntime: Seq[String]         = Seq.empty,
     initializeAtBuildtime: Seq[String]       = defaultBuildTimeInitClasses,
     mainClass: Option[String]                = None,
-    additionalCp: Seq[String]                = Seq(),
+    additionalCp: () => Seq[String]          = () => Seq(),
```

Comment:
`additionalCp` does not need to be a function. See my [other comment](https://github.com/enso-org/enso/pull/9866/files#r1592000092). This is an unnecessary complication.

Reply:
So far I am keeping as it is.